app/src/main/res/values/font_certs.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+</resources>
```

Comment:
Empty resource files don't have any effect, if I remember correctly. So the file should be safe to delete